solutions/nx-monorepo/babel.config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "babelrcRoots": ["*"]
+}
```

Comment:
Is this required in order to have Nx working even if the Next.js app doesn't use babel?

Reply:
Good question, let me play with that a little (this came from the Nx project generator)